pkg/config/nodetreemodel/config.go

```diff
@@ -222,16 +224,17 @@ func (c *ntmConfig) Set(key string, newValue interface{}, source model.Source) {
 	}
 
 	receivers := slices.Clone(c.notificationReceivers)
```

Comment:
We don't need to clone the receivers any more since we're holding the lock until the end of the function.